app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api::V1
+  class PasswordsController < Devise::PasswordsController
+
+    api :POST, '/users/password', 'Reset password'
+    def create
+      @user = User.find_by_email(user_params[:email])
+      if @user.present?
+        @user.send_reset_password_instructions
```

Comment:
@yunixon this will generate email with link to main application. But it should be link to new react application. Also you should add API to change password based on reset password token which you will call from react frontend application